### PR TITLE
Cache get account id by script hash

### PR DESCRIPTION
**Pull request by Flouse**
## Test: 8ms -> 0.4ms
```
web3_1          | [1] POST / 200 8.499 ms - 39
web3_1          | [1] request.method: gw_get_account_id_by_script_hash
web3_1          | [1] using cache: 0xdfb94d6794165b96668b4308607afc05790dc2110867d3370ceb8a412902e7b4 -> 0x2
web3_1          | [1] POST / 200 0.417 ms - 39
web3_1          | [1] request.method: gw_get_account_id_by_script_hash
web3_1          | [1] using cache: 0xdfb94d6794165b96668b4308607afc05790dc2110867d3370ceb8a412902e7b4 -> 0x2
web3_1          | [1] POST / 200 0.423 ms - 39
web3_1          | [1] request.method: gw_get_account_id_by_script_hash
web3_1          | [1] using cache: 0xdfb94d6794165b96668b4308607afc05790dc2110867d3370ceb8a412902e7b4 -> 0x2
web3_1          | [1] POST / 200 0.370 ms - 39
web3_1          | [1] request.method: gw_get_account_id_by_script_hash
web3_1          | [1] using cache: 0xdfb94d6794165b96668b4308607afc05790dc2110867d3370ceb8a412902e7b4 -> 0x2
web3_1          | [1] POST / 200 0.309 ms - 39
web3_1          | [1] request.method: gw_get_account_id_by_script_hash
web3_1          | [1] using cache: 0xdfb94d6794165b96668b4308607afc05790dc2110867d3370ceb8a412902e7b4 -> 0x2
web3_1          | [1] POST / 200 0.309 ms - 39
web3_1          | [1] request.method: gw_get_account_id_by_script_hash
web3_1          | [1] using cache: 0xdfb94d6794165b96668b4308607afc05790dc2110867d3370ceb8a412902e7b4 -> 0x2
web3_1          | [1] POST / 200 0.344 ms - 39
web3_1          | [1] request.method: gw_get_account_id_by_script_hash
web3_1          | [1] using cache: 0xdfb94d6794165b96668b4308607afc05790dc2110867d3370ceb8a412902e7b4 -> 0x2
web3_1          | [1] POST / 200 0.360 ms - 39
web3_1          | [1] request.method: gw_get_account_id_by_script_hash
web3_1          | [1] using cache: 0xdfb94d6794165b96668b4308607afc05790dc2110867d3370ceb8a412902e7b4 -> 0x2
web3_1          | [1] POST / 200 0.307 ms - 39
```

### CODE REVIEW

File: packages/api-server/src/methods/modules/gw.ts [hunk 1]
```diff
@@ -154,7 +163,18 @@ export class Gw {
    */
   async get_account_id_by_script_hash(args: any[]) {
     try {
-      const result = await this.rpc.gw_get_account_id_by_script_hash(...args);
+      const scriptHash = args[0];
+      let result = await this.gwCache.get(`${GW_RPC_KEY}_${scriptHash}`);
+      if (result !== null) {
```
Comment (classicalliu) [inline packages/api-server/src/methods/modules/gw.ts:168]:
```suggestion
      if (result != null) {
```

File: packages/api-server/src/methods/modules/gw.ts [hunk 2]
```diff
@@ -154,7 +163,18 @@ export class Gw {
...
+        console.debug(`using cache: ${scriptHash} -> ${result}`);
+        return result;
+      }
+
+      result = await this.rpc.gw_get_account_id_by_script_hash(...args);
+      if (result) {
```
Comment (classicalliu) [inline packages/api-server/src/methods/modules/gw.ts:174]:
```suggestion
      if (result != null) {
```

**Review by classicalliu — APPROVED** [review]

**Comment by classicalliu** [comment]
Run `yarn run fmt && yarn run lint` before commit to check code format and lint.

**Review by RetricSu — APPROVED** [review]

File: packages/api-server/src/cache/constant.ts [hunk 3]
```diff
@@ -2,3 +2,6 @@
 export const CACHE_EXPIRED_TIME_MILSECS = 5 * 60 * 1000; // milsec, default 5 minutes
 // limit redis store filter size
 export const MAX_FILTER_TOPIC_ARRAY_LENGTH = 20;
+
+// The Cache Key Prfixs
+export const GW_RPC_KEY = 'gwRPC';
```
Comment (classicalliu) [inline packages/api-server/src/cache/constant.ts:7]: Add new empty line at the end of file

**Pull request merged**